airflow/configuration.py

```diff
@@ -458,6 +464,12 @@ def run_command(command):
 scheduler_heartbeat_sec = 5
 authenticate = true
 max_threads = 2
+
+# Turn off scheduler backfills by setting this to false.
+# Default behavior is unchanged
+# Command Line Backfills still work, but the scheduler
+# will not do backfills if this is false.
+backfill_by_default = true
```

Comment:
that's strange, I read the configparser code and thought test config selectively overrides default config... will need to investigate